### PR TITLE
Update license for `2201.7.1`

### PR DESCRIPTION
**Pull request by rdulmina**
## Purpose
$title

### CODE REVIEW

File: ballerina/LICENSE [hunk 1]
```diff
@@ -17,169 +17,170 @@ ________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2
 commons-text-1.10.0.jar                                                                             bundle         apache2
 compiler-0.8.9.jar                                                                                  jar            apache2
-ballerina-rt-2201.7.0.jar                                                                           jar            apache2
-typedesc-2201.7.0.jar                                                                               jar            apache2
-shell-core-2201.7.0.jar                                                                             jar            apache2
+testerina-runtime-2201.7.1.jar                                             jar            apache2
```
Comment (malinthar) [inline ballerina/LICENSE:20]: Let's fix the indentation here and the other places

**Review by malinthar — CHANGES_REQUESTED** [review]

File: ballerina/LICENSE (hunk 1 above)
Comment (rdulmina) [reply ballerina/LICENSE]: Updated

**Review by dulajdilshan — APPROVED** [review]
LGTM!

File: ballerina/LICENSE [hunk 2]
```diff
@@ -372,15 +373,15 @@ epl2           Eclipse Public License Version 2.0
...
+mit            MIT License
+               http://www.opensource.org/licenses/mit-license.php
 bouncy         Bouncy Castle License
                http://www.bouncycastle.org/licence.html
 edl1           Eclipse Distribution License Version 1.0
                http://www.eclipse.org/org/documents/edl-v10.php
 mpl10          Mozilla Public License Version 1.0
-               http://www.mozilla.org/MPL/
\ No newline at end of file
+               http://www.mozilla.org/MPL/
```
Comment (malinthar) [inline ballerina/LICENSE:387]: Do we need these chnages? I think we can omit these

**Review by malinthar — CHANGES_REQUESTED** [review]

File: ballerina/LICENSE (hunk 2 above)
Comment (rdulmina) [reply ballerina/LICENSE]: Updated

**Review by malinthar — APPROVED** [review]
LGTM!

**Pull request merged**